internal/server/repository_workerauth.go

```diff
@@ -236,12 +236,9 @@ func (r *WorkerAuthRepositoryStorage) convertRootCertificate(ctx context.Context
 	if err != nil {
 		return nil, errors.Wrap(ctx, err, op)
 	}
-	rootCert.KeyId, err = databaseWrapper.KeyId(ctx)
-	if err != nil {
-		return nil, errors.Wrap(ctx, err, op)
-	}
-	rootCert.PrivateKey, err = encrypt(ctx, cert.PrivateKeyPkcs8, databaseWrapper)
-	if err != nil {
+
+	rootCert.PrivateKey = cert.PrivateKeyPkcs8
```

Comment:
What do you think about moving this above the GetWrapper call with the other assignments?